src/fe-text/irssi.c

```diff
@@ -313,6 +316,7 @@ int main(int argc, char **argv)
 	g_log_set_always_fatal(loglev);
 	textui_finish_init();
 	main_loop = g_main_new(TRUE);
+	dirty_check(); /* draw the screen prior to the first wait poll */
```

Comment:
I'd move this at the end of `textui_finish_init` where the other `dirty_check` is.
The comment isn't very helpful as-is as it just states where this piece of code is, should be accompanied by some words about /why/ we're redrawing the screen.